pennylane/ops/qubit.py

```diff
@@ -508,6 +554,89 @@ class CRot(Operation):
     grad_method = "A"
 
 
+class U2(Operation):
+    r"""U2(phi, lambda, wires)
+    U2 gate.
+
+    .. math::
+
+        U_2(\phi, \lambda) = \frac{1}{\sqrt{2}}\begin{bmatrix} 1 & -\exp(i \lambda)
+        \\ \exp(i \phi) & \exp(i (\phi + \lambda)) \end{bmatrix}
+
+    The :math:`U_2` gate is related to the single-qubit rotation :class:`Rot` and the
```

Comment:
Be good to add the mathematical names for the gates here, so the next line is clearer

Reply:
Mathematical names?

Reply:
in text they are called `Rot` and `PhaseShift`, but in the math block below, we use the names `R_\phi` and `R`. Would be good to make this consistent/clear

Reply:
Done